dagfactory/dagbuilder.py

```diff
@@ -723,6 +723,45 @@ def process_file_with_datasets(file: str, datasets_conditions: str) -> Any:
             datasets_uri = utils.get_datasets_uri_yaml_file(file, list(dataset_map.keys()))
             return [Dataset(uri) for uri in datasets_uri]
 
+    @staticmethod
+    def _get_schedule_obj(dag_params: Dict[str, Any]):
+        schedule = dag_params.get("schedule")
```

Comment:
I wonder if we should also get `schedule_interval` if `schedule` doesn't exist. Could be nice to use this as a compatibility layer

Reply:
Yes, we'll take care of that. This functionality is specific to AF3. If AF2 is installed in the environment, the previous logic will continue to work as expected.